Update docs related to styles (#62)
asyncapi/asyncapi-react: docs/configuration/config-modification.md

  This field contains configuration responsible for rendering specific parts of the AsyncApi component.
  All fields are set to `true` by default.

- **expand?: Partial<ExpandConfig>**

>   This field contains configuration responsible for initial expanding specific parts of the AsyncApi component.

Reviewer: ```suggestion
  This field contains configuration responsible for expanding specific parts of the AsyncApi component automatically.
```

>   `root` means root component for specific parts of the AsyncApi component. `Elements` means elements inside `root` component.

Reviewer: ```suggestion
  `root` refers to a root component for specific parts of the AsyncAPI component, and `Elements` refers to elements inside the `root` component.
```

>   By default `expand.channels.root` is set to `true`.

Reviewer: ```suggestion
  By default, `expand.channels.root` is set to `true`.
```

- **showErrors?: boolean**

  This field turns on or off the option displaying validation or parsing errors that show at the top of the component.
  This field is set to `true` by default.

- **parserOptions?: ParserOptions**

  This field contains configuration for [`asyncapi-parser`](https://github.com/asyncapi/parser). See available options [here](https://github.com/asyncapi/parser-js/blob/master/API.md#parser).
  This field is set to `null` by default.

## Examples

See exemplary component configuration in TypeScript and JavaScript.

### TypeScript

```tsx
import * as React from "react";
import { render } from "react-dom";
import AsyncApiComponent, { ConfigInterface } from "asyncapi-react";

import { schema } from "./mock";

const config: Partial<ConfigInterface> = {
  show: {
    schemas: false
  },
  showErrors: false
};

const App = () => <AsyncApiComponent schema={schema} config={config} />;

render(<App />, document.getElementById("root"));
```

> ### JavaScript

Reviewer 2: I know we love TS, but pure JS example should be first, as it was before 😢

Author: Nope :P
